dbt_project.yml

```
@@ -19,9 +19,13 @@ vars:
     jira_status: "{{ source('rag_jira', 'status') }}"
     jira_priority: "{{ source('rag_jira', 'priority') }}"
 
-models:
-  unified_rag:
-    intermediate:
-      +materialized: ephemeral
-    staging:
-      +materialized: view
\ No newline at end of file
+    # Hubspot Sources
+    hubspot_engagement_note: "{{ source('rag_hubspot', 'engagement_note') }}"
+    hubspot_engagement_email: "{{ source('rag_hubspot', 'engagement_email') }}"
+    hubspot_engagement_company: "{{ source('rag_hubspot', 'engagement_company') }}"
+    hubspot_engagement_contact: "{{ source('rag_hubspot', 'engagement_contact') }}"
+    hubspot_engagement_deal: "{{ source('rag_hubspot', 'engagement_deal') }}"
+    hubspot_company: "{{ source('rag_hubspot', 'company') }}"
+    hubspot_contact: "{{ source('rag_hubspot', 'contact') }}"
+    hubspot_deal: "{{ source('rag_hubspot', 'deal') }}"
+    hubspot_owner: "{{ source('rag_hubspot', 'owner') }}"
```

Just making sure the models config is retained after merge
```suggestion
    hubspot_owner: "{{ source('rag_hubspot', 'owner') }}"
    
models:
  unified_rag:
    intermediate:
      +materialized: ephemeral
    staging:
      +materialized: view
```